sdk/communication/Azure.Communication.NetworkTraversal/api/Azure.Communication.NetworkTraversal.netstandard2.0.cs

```diff
@@ -15,29 +15,22 @@ public partial class CommunicationRelayClient
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Communication.NetworkTraversal.CommunicationRelayConfiguration>> GetRelayConfigurationAsync(Azure.Communication.CommunicationUserIdentifier communicationUser = null, Azure.Communication.NetworkTraversal.RouteType? routeType = default(Azure.Communication.NetworkTraversal.RouteType?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
     public partial class CommunicationRelayClientOptions : Azure.Core.ClientOptions
     {
-        public CommunicationRelayClientOptions(Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions.ServiceVersion version = Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions.ServiceVersion.V2021_10_08_preview) { }
+        public CommunicationRelayClientOptions(Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions.ServiceVersion version = Azure.Communication.NetworkTraversal.CommunicationRelayClientOptions.ServiceVersion.V2022_02_01) { }
         public enum ServiceVersion
         {
-            V2021_10_08_preview = 1,
+            V2022_02_01 = 1,
         }
     }
     public partial class CommunicationRelayConfiguration
     {
-        internal CommunicationRelayConfiguration() { }
-        public System.DateTimeOffset ExpiresOn { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<Azure.Communication.NetworkTraversal.CommunicationIceServer> IceServers { get { throw null; } }
-    }
-    public static partial class NetworkTraversalModelFactory
-    {
-        public static Azure.Communication.NetworkTraversal.CommunicationIceServer CommunicationIceServer(System.Collections.Generic.IEnumerable<string> urls = null, string username = null, string credential = null, Azure.Communication.NetworkTraversal.RouteType routeType = default(Azure.Communication.NetworkTraversal.RouteType)) { throw null; }
-        public static Azure.Communication.NetworkTraversal.CommunicationRelayConfiguration CommunicationRelayConfiguration(System.DateTimeOffset expiresOn = default(System.DateTimeOffset), System.Collections.Generic.IEnumerable<Azure.Communication.NetworkTraversal.CommunicationIceServer> iceServers = null) { throw null; }
+        public CommunicationRelayConfiguration(System.DateTimeOffset? expiresOn, System.Collections.Generic.IEnumerable<Azure.Communication.NetworkTraversal.CommunicationIceServer> iceServers) { }
+        public System.DateTimeOffset? ExpiresOn { get { throw null; } set { } }
```

Comment:
Why has `ExpiresOn` become nullable?

Reply:
Put it back to how it was after discussing offline.